katello/katello.spec

```diff
@@ -26,6 +26,10 @@ Source9:    qpid-core-dump
 Source10:   katello-clean-empty-puppet-environments
 Source11:   katello-change-hostname
 Source12:   katello-repository-publish-check
+Source13:   restore.rb
```

Comment:
at this point I wonder if we should create a proper katello_utils gem, put all the real code there, and just package require that?

Reply:
Right, I remember hearing that. I guess it highly depends on the time-schedule then?
Creating a gem is (IMHO) pretty easy and cheap, unless we're moving to foreman-maintain in 2 wks or something.
The code requires a class-ification for either path (pure class-ification, a k_u gem or f_m gem) and if make a gem now, the move to foreman-maintain is just a rename of the files away.

Reply:
@evgeni I see your point and it makes sense. However I don't expect the move to be just a rename. `foreman-maintain` is supposed to work on all the scenarios we have (foreman, katello, various combination of the plugins, upstream, downstream). f-m as a framework introduces `features`and behaves on their presence or absence. Some of the logic is already there some will need to be extended. I expect the code here to get restructured to match f-m's layout. With that in mind I just wanted to rise discussion if further classification still makes sense.